accesshandler/pkg/runtime/lambda/revoke_grant.go

```diff
@@ -57,11 +56,11 @@ func (r *Runtime) RevokeGrant(ctx context.Context, grantID string, revoker strin
 		return nil, err
 	}
 
-	if grant.Status == types.ACTIVE {
```

Comment:
I'm curious about this.
From my understanding, the revoke endpoint handles both cancelling, 
cancelling is when access is not provisioned in the target but a stepfunction is running, the grant is in the pending state.

A grant in the active state means there is a stepfunction and access is provisioned in the target.
So if revoking is failing because of this if statement, its more likely to me that the grant is never becoming Active rather than this being a bug

Reply:
Thats what I thought too. But  since we don't have the concept of a GRANT in the database anymore, `grant` in this case is just the input to the state function. Which doesn't change along the lifecycle (I think) so would always be PENDING.
I guess the error here was that it was looking up the state of the grant from the wrong place?